tests/test_jointcal_hsc.py

```diff
@@ -100,11 +103,26 @@ def test_jointcalTask_2_visits_simple(self):
 
     def test_jointcalTask_2_visits_simple_gen3(self):
         """Test gen3 butler jointcal."""
-        queryString = "instrument='HSC' and tract=9697 and skymap='hsc_rings_v1' and band='r'"
-        queryString += f" and visit in ({self.all_visits[0]},{self.all_visits[1]})"
-        configOptions = ["astrometryModel=simple", "photometryModel=simpleFlux"]
-        self._runGen3Jointcal("lsst.obs.subaru.HyperSuprimeCam", "HSC", queryString,
-                              configOptions=configOptions)
+        configOptions = {"astrometryModel": "simple", "photometryModel": "simpleFlux"}
+        where = f" and visit in ({self.all_visits[0]},{self.all_visits[1]})"
+
+        metrics = {'astrometry_collected_refStars': 568,
+                   'photometry_collected_refStars': 6485,
+                   'astrometry_prepared_refStars': 137,
+                   'photometry_prepared_refStars': 1609,
+                   'astrometry_matched_fittedStars': 2070,
+                   'photometry_matched_fittedStars': 2070,
+                   'astrometry_prepared_fittedStars': 989,
+                   'photometry_prepared_fittedStars': 1731,
+                   'astrometry_prepared_ccdImages': 6,
+                   'photometry_prepared_ccdImages': 6,
+                   'astrometry_final_chi2': 835.473,
+                   'astrometry_final_ndof': 1918,
+                   'photometry_final_chi2': 4997.62,
+                   'photometry_final_ndof': 2188
+                   }
+        self._runGen3Jointcal("lsst.obs.subaru.HyperSuprimeCam", "HSC", whereSuffix=where,
```

Comment:
I must be missing something because it seems like you should still have to specify a tract, etc., but I don't see how that's done in the new way

Reply:
That is in the `where` that is passed to `setUp_base()` (along with instrument, skymap, and band). The `whereSuffix` kwarg of `_runGen3Jointcal` gets appended to that.